client/Windows/wf_floatbar.c

```diff
@@ -25,6 +25,7 @@
 #include "wf_client.h"
 #include "wf_floatbar.h"
 #include "wf_gdi.h"
+#pragma comment( lib, "Msimg32.lib" )
```

Comment:
Yes, according to the docs for GradientFill() et al.  Are you really supporting versions of Windows prior to that?